warehouse/api/openapi.yaml

```diff
@@ -0,0 +1,112 @@
+openapi: 3.0.0
+info:
+  title: Warehouse Management API
+  description: "**BETA** API for managing warehouse interactions"
+  version: 0.0.0
+servers:
+  - url: http://localhost
+    description: Local development server
```

Comment:
TODO: Is it possible to dynamically create these servers based on pypi/testpypi/localhost?

Reply:
It is! Removing them does the trick.